src/components/environments-summary/environment-ingress.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { faLink } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+import usePollComponents from './use-poll-components';
+
+const URL_VAR_NAME = 'RADIX_PUBLIC_DOMAIN_NAME';
+const MAX_DISPLAY_NR_COMPONENT = 3;
+
+const EnvironmentIngress = ({ appName, deploymentName }) => {
+  const [componentsPollState] = usePollComponents(appName, deploymentName);
+  let components =
```

Comment:
This is a very long if statement.
Suggestion to make it easier to read:
```
let components = [];
if (componentsPollState && componentsPollState.data) {
   components = componentsPollState.data.filter(component => component.variables[URL_VAR_NAME]);
}
```
...Unless this turns into a blinking anomaly due to component reloading, and you have to set the correct value when declaring the variable in order to avoid that blinking.

Reply:
updated with your suggestion